services/core-api/app/api/mines/mine/models/mine.py

```diff
@@ -23,44 +23,60 @@ class Mine(AuditMixin, Base):
     __tablename__ = 'mine'
```

Comment:
The majority of the changes on the BE files are formatting.

Reply:
I wonder why. We should try to get the team using a consistent formatted :(
In VS Code you can do "Ctrl + K, Ctrl + S" to save without applying formatting.

Reply:
I thought we did have consistent formatting in our settings.JSON? Im wondering if mine got removed for some reason